helm/polaris/build.gradle.kts

```diff
@@ -0,0 +1,245 @@
+  """
+          .trimIndent(),
+      )
+
+    inputs.files(
+      fileTree(projectDir) { include("Chart.yaml", "values.yaml", "ci/*.yaml", "templates/**/*") }
+    )
+
+    outputs.file(outputFile)
+  }
+
+val helmUnitTest by
+  tasks.registering(Exec::class) {
+    group = "verification"
+    description = "Run Helm unit tests"
+
+    workingDir = projectDir
+
+    val outputFile = helmTestReportsDir.get().file("helm-unit/test.log")
+
+    // Install the plugin if not already installed, then run tests
+    commandLine =
+      listOf(
+        "sh",
+        "-c",
+        """
+    mkdir -p ${outputFile.asFile.parent}
+    helm plugin install https://github.com/helm-unittest/helm-unittest.git 2>/dev/null || true
+    helm unittest . > ${outputFile.asFile} 2>&1
+  """
+          .trimIndent(),
+      )
+
+    inputs.files(
+      fileTree(projectDir) { include("Chart.yaml", "values.yaml", "templates/**/*", "tests/**/*") }
+    )
+
+    outputs.file(outputFile)
+  }
+
+val chartTestingLint by
+  tasks.registering(Exec::class) {
+    group = "verification"
+    description = "Run chart-testing (lint)"
+
+    workingDir = rootProject.projectDir
+
+    val outputFile = helmTestReportsDir.get().file("ct/lint.log")
+
+    commandLine =
+      listOf(
+        "sh",
+        "-c",
+        """
+    mkdir -p ${outputFile.asFile.parent}
+    ct lint --debug --charts ./helm/polaris > ${outputFile.asFile} 2>&1
+  """
+          .trimIndent(),
+      )
+
+    inputs.files(
+      fileTree(projectDir) { include("Chart.yaml", "values.yaml", "ci/*.yaml", "templates/**/*") }
+    )
+
+    outputs.file(outputFile)
+  }
+
+// Task to build Docker images in minikube's Docker environment
+val buildMinikubeImages by
+  tasks.registering(Exec::class) {
+    group = "build"
+    description = "Build Polaris Docker images in minikube's Docker environment"
+
+    workingDir = rootProject.projectDir
+
+    // Output: file containing SHA digest of built image
+    val outputFile = helmTestReportsDir.get().file("minikube-images.sha256")
+
+    commandLine =
+      listOf(
+        "sh",
+        "-c",
+        """
+    # Check if minikube is running
+    if ! minikube status >/dev/null 2>&1; then
+      echo "Minikube is not running. Starting minikube..."
+      minikube start
+    fi
+
+    # Set up docker environment and build images
+    eval $(minikube -p minikube docker-env)
+
+    # Build server image
+    docker build -t apache/polaris:latest \
+      -f runtime/server/src/main/docker/Dockerfile.jvm \
+      runtime/server
+
+    # Capture image digest
+    mkdir -p ${outputFile.asFile.parent}
+    {
+      docker inspect --format='{{.Id}}' apache/polaris:latest
+    } > ${outputFile.asFile}
+  """
+          .trimIndent(),
+      )
+
+    dependsOn(":polaris-server:quarkusBuild")
+
+    inputs.dir(rootProject.file("runtime/server/build/quarkus-app"))
+    inputs.file(rootProject.file("runtime/server/src/main/docker/Dockerfile.jvm"))
```

Comment:
Hm - better use another Gradle configuration pair. Referencing other projects leads to issues.